src/game_api/script/usertypes/entity_casting_lua.cpp

```diff
@@ -1,5 +1,4 @@
 #include "entity_casting_lua.hpp"
-
```

Comment:
There's no need to remove empty lines, those kinds of things are fixed by clang-format and we don't enforce it. So whoever wrote this file might've had a good reason to put the empty line. Afaik empty lines after the corresponding header include is very standard.